CLAUDE.md

```
@@ -32,9 +32,9 @@ const fetchAlerts = async () => {
 - NIEMALS andere CSS-Frameworks oder CSS-in-JS mischen
 - Bei UI-Änderungen IMMER Tailwind-Klassen und Headless UI-Komponenten verwenden
 - **Tailwind Plus (TailwindUI):** Premium-Komponenten
-  - IMMER beim User anfragen!
-  - Der User muss die Komponenten manuell von TailwindUI kopieren und bereitstellen
-  - NIEMALS selbst TailwindUI-Komponenten erfinden oder raten
+    - IMMER beim User anfragen!
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)